--- /home/htj/src/python-suds-0.4/suds/sax/date.py	2010-09-08 15:54:12.000000000 +0200
+++ date.py	2011-09-06 11:59:17.152679075 +0200
@@ -346,7 +346,10 @@
     
     pattern = re.compile('([zZ])|([\-\+][0-9]{2}:[0-9]{2})')
     
-    LOCAL = ( 0-time.timezone/60/60 )
+    if time.daylight:
+        LOCAL = ( 0 - time.altzone/3600 )
+    else:
+        LOCAL = ( 0 - time.timezone/3600 )
 
     def __init__(self, offset=None):
         if offset is None:
